network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -1027,6 +1030,18 @@ def localized_related_products(self):
         related_products = localize_queryset(related_products, preserve_order=True)
         return related_products.specific()
 
+    @property
+    def preview_related_products(self) -> list:
+        """
+        Fetches related product updates for CMS page previews.
+        """
+        related_products = orderables.get_related_items(
+            self.related_product_pages.all(), "related_product", order_by="sort_order"
+        )
+        # This method, while inneficient, is neccesary as we are using it to render
+        # any related_product updates that are stored in memory, but not yet saved in the DB.
+        return [product.localized for product in related_products]
```

Comment:
Now that I'm thinking about it, the "related_products" selection only occurs in the English/default version. The other versions are synced to the default location, so they don't have a "preview" window. I think it's safe to drop the `.localized` here